vsutil.py

```diff
@@ -150,19 +160,19 @@ def frame2clip(frame: vs.VideoFrame, *, enforce_cache=True) -> vs.VideoNode:
     return result
 
 
-def get_w(height: int, aspect_ratio: float = 16 / 9, only_even: bool = True) -> int:
+def get_w(height: int, /, aspect_ratio: float = 16 / 9, *, only_even: bool = True) -> int:
```

Comment:
I’d prefer not having height as positional-only here.
It feels like the kind of parameter to me that could be a named argument for clarity in some scripts, and I don’t want to break those.